Sources/Microcharts/Layouts/LineChart.cs

```diff
@@ -134,13 +149,25 @@ protected void DrawLine(SKCanvas canvas, SKPoint[] points, SKSize itemSize, IEnu
                                     using (var textPaint = new SKPaint
                                     {
                                         Style = SKPaintStyle.StrokeAndFill,
-                                        Color = TooltipTextColor == default(SKColor) ? entry.Color : TooltipTextColor,
+                                        Color = TooltipTextColor,
                                         TextAlign = SKTextAlign.Center,
-                                        TextSize = TooltipTextSize,
-                                        StrokeWidth = LineSize
+                                        TextSize = TooltipTextSize
                                     })
                                     {
-                                        canvas.DrawText(entry.Label, _tooltipPoint.X, _tooltipPoint.Y - _touchRadius, textPaint);
+                                        var topBottomMargin = TooltipTextSize;
+                                        var tooltipTextYPosition = _tooltipPoint.Y - TooltipYOffset - topBottomMargin;
+                                        var textPath = textPaint.GetTextPath(entry.Label, _tooltipPoint.X, tooltipTextYPosition);
```

Comment:
Have you tried [MeasureText(string, ref rect)](https://docs.microsoft.com/en-us/dotnet/api/skiasharp.skpaint.measuretext?view=skiasharp-1.68.1#SkiaSharp_SKPaint_MeasureText_System_String_SkiaSharp_SKRect__) ? My guess is that it should be more performant than calculating the path.

Reply:
As I get it (I have not tested yet) MeasureText(string, ref rect) returns the bounts in the ref rect parameter. Judging by this example https://docs.microsoft.com/en-us/xamarin/xamarin-forms/user-interface/graphics/skiasharp/basics/text
 // Find the text bounds
SKRect textBounds = new SKRect();
textPaint.MeasureText(str, ref textBounds);

Reply:
Still not sure it would work - I'd prefer to leave it like it is now as I've tested it and it works. Also this code is executed rarely and for only 1 tooltip at a time, so performance from this change wouldn't really make much of a difference.

Reply:
That's true. Ok.